sdks/python/scripts/run_integration_test.sh

```diff
@@ -194,8 +194,10 @@ if [[ -z $PIPELINE_OPTS ]]; then
   fi
 
   # Install test dependencies for ValidatesRunner tests.
-  echo "pyhamcrest" > postcommit_requirements.txt
-  echo "mock" >> postcommit_requirements.txt
+  # pyhamcrest==1.10.0 doesn't work on Py2.
```

Comment:
I don't like adding yet another location with version numbers. I wish we a single SoT for versioning.